hub/resourceoutlines/policy/v1/poddisruptionbudgets.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: meta.k8s.appscode.com/v1alpha1
+kind: ResourceOutline
+metadata:
+  creationTimestamp: null
+  labels:
+    k8s.io/group: policy
+    k8s.io/kind: PodDisruptionBudget
+    k8s.io/resource: poddisruptionbudgets
+    k8s.io/version: v1
+  name: policy-v1-poddisruptionbudgets
+spec:
+  defaultLayout: true
+  pages:
+  - name: Overview
+    sections:
+    - blocks:
+      - kind: Block
+        name: policy-v1-poddisruptionbudgets
```

Comment:
For simple blocks like this, we should just put this here directly.

Reply:
done